golang/cosmos/x/swingset/types/params.go

```diff
@@ -163,3 +163,75 @@ func validateQueueMax(i interface{}) error {
 	}
 	return nil
 }
+
+// UpdateParams appends any missing params, configuring them to their defaults,
+// then returning the updated params or an error.
```

Comment:
Add that existing params will not be modified, regardless of their value, and regardless of whether the parameter appears in the defaults.